components/CommentCount.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString (count) {
+    if (count == undefined) { count = 0 }
```

Comment:
You should define `propTypes` for this component, and add the `count` prop. Since you're worried about `count` being undefined, maybe you could define `defaultProps` and set it to 0 by default?